etip/restful_api/serializers.py

```diff
@@ -23,10 +23,21 @@ class Meta:
           'is_in_exodus',
           'code_signature',
           'network_signature',
+          'api_key_ids',
           'website',
           'maven_repository',
           'group_id',
           'artifact_id',
           'gradle',
+          'created',
+          'updated',
+          'creation_date',
+          'capability',
+          'advertising',
+          'analytic',
+          'network',
```

Comment:
**issue**: I'm having doubts about these fields, as they are lists and I'm not sure how they would be exposed through the API. It definitely needs more tests if we want to expose them.

Reply:
I added a test for those.  They are all name-only subclasses of `trackers.models.Category`, so I think one test should cover them all.